packages/decoder/lib/decoders.ts

```diff
@@ -271,36 +266,47 @@ export class WireDecoder {
       result = decoder.next(response);
     }
     //at this point, result.value holds the final value
-    const decoding = result.value;
-
-    return {
-      ...transaction,
-      decoding
-    };
+    return result.value;
   }
 
   /**
    * **This method is asynchronous.**
    *
-   * Takes a Web3
-   * [Log](https://web3js.readthedocs.io/en/v1.2.1/web3-eth.html#eth-getpastlogs-return)
-   * object and returns a copy of that object but with an additional decodings
-   * field.  This field holds an array of [[LogDecoding|LogDecodings]]; see the
-   * documentation on [[DecodedLog]] for more.
+   * Takes a [[Log]] object and decodes it.  Logs can be ambiguous, so this so
+   * this function returns an array of [[LogDecoding|LogDecodings]].
+   *
+   * Note that logs are decoded in strict mode, so (with one exception) none of the decodings should
```

Comment:
Is "strict mode" defined somewhere?

Reply:
Ooh, good catch.  No, I don't think it is.  Of course this isn't a new problem with the docs (this is just moved from elsewhere), but it hadn't previous occurred to me that it *is* a problem.  Thanks.  Um, I'm thinking maybe just merge now, fix later?